misc/de-metas-common/de-metas-common-util/src/main/java/de/metas/common/util/Constants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * #%L
+ * de-metas-common-util
+ * %%
+ * Copyright (C) 2024 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.common.util;
+
+public final class Constants
```

Comment:
IMHO these 2 constants are really not needed... more, they are also misleading

* NULL_REPO_ID used in only one place and that's a test!  ... all the other RepoIdAware impl are using `-1`
* NULL="" - so empty is the new null? :))))

Reply:
Always thought why don't we have a constant for -1, just in case we want to change this at some point
and Empty String is kind of null all over the application (Trim Blank to null is used in many places), so I was thinking, why not name these 2 we use all over the place, but ok as long as it isn't used in more places directly I don't add it like this